### PR TITLE
Rename application to release, fleet or app where appropriate

### PR DESCRIPTION
**Pull request by tmigone**
Change-type: patch
Signed-off-by: Tomás Migone <tomas@balena.io>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
@tmigone Just to confirm, I agree with removing the screenshot. But, did you forget to replace that screenshot or are we removing it? 
I think removing it was the right choice, it didn't provide much value. 

I have updated your PR with more instances of applications that I found.

File: README.md [hunk 1]
```diff
@@ -158,15 +158,12 @@ A user can grant access to support agents by selecting the device from the
 Dashboard, and then selecting the 'Actions' tab in the left-hand sidepanel.
```
Comment (chrisys) [inline README.md:158]:
```suggestion
Dashboard, and then selecting the 'Settings' tab in the left-hand sidepanel.
```

File: README.md [hunk 2]
```diff
@@ -158,15 +158,12 @@ A user can grant access to support agents by selecting the device from the
...
 Scrolling down the Actions page will show a list of actions, with the 'Grant
```
Comment (chrisys) [inline README.md:159]:
```suggestion
Scrolling down the Settings page will show a list of actions, with the 'Grant
```

this whole part is unnecessarily wordy - e.g. do we really need to say 'scrolling the page will show a list', but perhaps a change for another time!

File: README.md [hunk 3]
```diff
@@ -158,15 +158,12 @@ A user can grant access to support agents by selecting the device from the
...
-support has been granted, the Dashboard will look something like this:
+determine the amount of time that support agents are allowed access for.
 
-![Granted Support Access](https://github.com/balena-io/debugging-masterclass/blob/master/resources/black-mountain-granted.png?raw=true)
-
-**Note:** It's also possible for a user to grant support for an entire
-application by selecting the application's Dashboard page and going through
+**Note:** It's also possible for a user to grant support to an entire
+fleet by selecting the fleet's Dashboard page and going through
 the same process (selecting 'Actions' and then 'Grant Support Access'). Granting
```
Comment (chrisys) [inline README.md:165]:
```suggestion
the same process (selecting 'Settings' and then 'Grant Support Access'). Granting
```

File: README.md [hunk 4]
```diff
@@ -656,17 +653,16 @@ them:
...
-    self-healing (via monitoring) of those applications, variables (application/
-    device/fleet) and exposure of these services to applications via an
-    endpoint.
-* `dbus.service` - The DBus daemon socket can be used by applications by
-    applying the `io.balena.features.dbus` label, which exposes it in-container.
-    This allows applications to control several host OS features, including the
-    Network Manager.
+    the management of releases, including downloading updates for and
+    self-healing (via monitoring) of those release services, variables (
+    device/fleet) and exposure of these services to apps via an endpoint.
```
Comment (chrisys) [inline README.md]:
```suggestion
    device/fleet) and exposure of these services to containers via an API endpoint.
```

File: README.md [hunk 5]
```diff
@@ -694,10 +690,10 @@ may have occurred (or which services may have failed causing a reboot).
 
 To alleviate this, balenaOS allows persistent journals (logs) to be enabled by
 customers. The easiest way to achieve this is to go the the Dashboard URL for
-the application or device in question, select 'Fleet Configuration' or 'Device
+the fleet or device in question, select 'Configuration' or 'Device
```
Comment (chrisys) [inline README.md:696]:
```suggestion
the fleet or device in question, select '
```

Likelihood is we can drop the word 'device' from this menu option soon, and I think this is clear either way anyway.

File: README.md [hunk 6]
```diff
@@ -694,10 +690,10 @@ may have occurred (or which services may have failed causing a reboot).
...
 Configuration' respectively, and then select 'Activate' on 'Enable persistent
```
Comment (chrisys) [inline README.md:697]:
```suggestion
Configuration', and then select 'Activate' on 'Enable persistent
```

File: README.md [hunk 7]
```diff
@@ -767,7 +763,7 @@ Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [16B blob data]
 Jan 13 11:05:20 dee2945 e374bbb9ddd4[765]: [1B blob data]
 ```
 
-If you're not examining the application's services then this output is fine, but
+If you're not examining the app's services then this output is fine, but
```
Comment (chrisys) [inline README.md:769]:
```suggestion
If you're not examining services then this output is fine, but
```

File: README.md [hunk 8]
```diff
@@ -867,7 +862,7 @@ root@f34c2e9:~#
 ```
 
 Your device should be connected, bring up the Dashboard page for the device.
-It should be 'Online' and running the pushed application code.
+It should be 'Online' and running the pushed release code.
```
Comment (chrisys) [inline README.md:868]:
```suggestion
It should be 'Online' and running the appropriate release.
```

File: README.md [hunk 9]
```diff
@@ -1257,7 +1252,7 @@ balena VPN. This include various data such as the device status, actions,
 SSH access, public URLs etc.
 
 Initially, the `os-config.service` unit requests a block of configuration data
-from the API, once the device has been registered against the application. Let's
+from the API, once the device has been registered against the fleet. Let's
```
Comment (chrisys) [inline README.md:1258]:
```suggestion
from the API, once the device has joined the fleet. Let's
```

File: README.md [hunk 10]
```diff
@@ -2068,7 +2063,7 @@ tempting to attribute them to balenaEngine instead of the actual underlying
 issue. A couple of examples
 of issues which are misattributed to :
 
-* Failure to download application service updates - usually because there is an
+* Failure to download release service updates - usually because there is an
```
Comment (chrisys) [inline README.md:2069]:
```suggestion
* Failure to download new releases - usually because there is an
```

File: README.md [hunk 11]
```diff
@@ -3225,9 +3220,9 @@ of free space on a device, or that of SD card corruption.
 
 A media partition that is full can cause issues such as the following:
 
-* Failure to download application updates, or failure to start new/updated
+* Failure to download release updates, or failure to start new/updated
```
Comment (chrisys) [inline README.md:3226]:
```suggestion
* Failure to download new releases, or failure to start new/updated
```

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**